### PR TITLE
Added EOL banner for Google Shopping ads Channel

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) adds EOL banner to Google Shopping ads install and release notes topics.

## Affected DevDocs pages

- https://devdocs.magento.com/extensions/google-shopping-ads/
- https://devdocs.magento.com/extensions/google-shopping-ads/release-notes/

### CODE REVIEW

File: src/extensions/google-shopping-ads/index.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ group: extensions
 title: Install Google Shopping ads Channel
 ---
 
+{:.bs-callout .bs-callout-info}
```
Comment (jeff-matthews) [inline src/extensions/google-shopping-ads/index.md]:
```suggestion
{:.bs-callout-warning}
```

File: src/extensions/google-shopping-ads/release-notes/index.md [hunk 2]
```diff
@@ -3,6 +3,9 @@ group: extensions
 title: Google Shopping ads Channel Release Notes
 ---
 
+{:.bs-callout .bs-callout-info}
```
Comment (jeff-matthews) [inline src/extensions/google-shopping-ads/release-notes/index.md]:
```suggestion
{:.bs-callout-warning}
```

File: src/extensions/google-shopping-ads/index.md (hunk 1 above)
Comment (hickskenh) [reply src/extensions/google-shopping-ads/index.md]: agreed to stay blue to be consistent with merchdocs banner

File: src/extensions/google-shopping-ads/release-notes/index.md (hunk 2 above)
Comment (hickskenh) [reply src/extensions/google-shopping-ads/release-notes/index.md]: agreed to stay blue to be consistent with merchdocs banner

File: src/extensions/google-shopping-ads/index.md (hunk 1 above)
Comment (dshevtsov) [reply src/extensions/google-shopping-ads/index.md]: not sure what consistency do you mean
it's warning on merchdocs:
https://github.com/magento/merchdocs/blob/master/src/sales-channels/google-ads/google-ad-channel.md

File: src/extensions/google-shopping-ads/release-notes/index.md (hunk 2 above)
Comment (dshevtsov) [reply src/extensions/google-shopping-ads/release-notes/index.md]: same as on my previous comment

**Comment by hickskenh** [comment]
running tests

**Comment by hickskenh** [comment]
running tests

File: src/extensions/google-shopping-ads/index.md (hunk 1 above)
Comment (hickskenh) [reply src/extensions/google-shopping-ads/index.md]: the header bar that has this same text is in a blue bar at the top of every page of the Google section. and when shown to the product manager in the orange, he asked why it was different, so i changed it back to blue.

**Pull request merged**